src/status_im/common/pdf_viewer/view.cljs

```diff
@@ -0,0 +1,23 @@
+(ns status-im.common.pdf-viewer.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [react-native.pdf-viewer :as pdf-viewer]
+    [utils.re-frame :as rf]))
+
+(defn- on-close
+  []
+  (rf/dispatch [:navigate-back]))
```

Comment:
Can use `navigate-back` function from `status-im.common.events-helper`

Reply:
I was searching for that function in navigation ns 🤦  and ended up adding it to the file. I will update it from `status-im.common.events-helper`. Thanks 🙏